Makefile

```diff
@@ -7,7 +7,16 @@ validate:
 	terraform fmt --recursive
 
 docs:
+	rm -rf modules/*/.terraform modules/*/.terraform.lock.hcl
+	rm -rf examples/*/.terraform examples/*/.terraform.lock.hcl
 	terraform-docs -c .terraform-docs.yml .
+	cd examples/gcp_bigquery_big_lake_table/; terraform-docs markdown . --output-file README.md --output-mode inject
```

Comment:
Can we have a .terraform-docs.yaml inside each of the examples/**/ folder and use recursive functionality of terraform-docs?

https://terraform-docs.io/user-guide/configuration/recursive/

Reply:
We're already using `recursive` but it seems to only support one folder (in this case it's set to `modules`) - or do you mean something else?

```
recursive:
  enabled: true
  path: modules
output:
  file: README.md
  mode: inject
formatter: "markdown"
```